src/__tests__/a11yAPI.test.tsx

```diff
@@ -70,51 +69,51 @@ function Section() {
   );
 }
 
-test('getByA11yLabel, queryByA11yLabel, findByA11yLabel', async () => {
-  const { getByA11yLabel, queryByA11yLabel, findByA11yLabel } = render(
+test('getByLabelText, queryByLabelText, findByLabelText', async () => {
+  const { getByLabelText, queryByLabelText, findByLabelText } = render(
```

Comment:
I wonder why we are changing alias for less `a11y` name despite it is `a11yAPI` test. Is there any argument for it or only preference?

Reply:
I guess I should actually rename the test. It was discussed a bit earlier, the point was to remove the aliases, currently there was `ByA11yLabel`, `ByAccessibilityLabel`, `ByLabelText`. RTL web only supports `ByLabelText`, so here we would align the API.